userguide/snippets/alertevents.rst

```diff
@@ -20,57 +20,56 @@ the system is warning that the S.M.A.R.T. service is not running.
    Example Alert Message
 
 
-Informational messages have a green :guilabel:`OK`, warning messages
-flash yellow, and messages requiring attention are listed as a red
-:guilabel:`CRITICAL`. CRITICAL messages are also emailed to the root
+Alert messages have a blue and white :literal:`i` icon. Messages
```

Comment:
With themes, this will not always be blue and white.  Just say an icon.  And rather than include it inline, let's add a screenshot.